codebases/circulaw.html

```diff
@@ -0,0 +1,111 @@
+---
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: The Foundation for Public Code <info@publiccode.net>
+layout: codebase
+title: Codebases we work with - CircuLaw
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s2">
+      <div class="cb-logo"><img src="../photos/codebases/circulaw/CircuLaw-logo.png" alt="CircuLaw"></div>
+      <h1>CircuLaw</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>CircuLaw is a knowledge platform dedicated to enabling the transition to a circular economy by identifying opportunities in current law to support a circular future.</p>
```

Comment:
Would add another para with this info to give broder aims.

 `<p>Amidst the escalating climate crisis, the global community is under significant pressure to transition towards a more sustainable and circular economy. With the European Union’s Circular Economy Action Plan and the Dutch government's commitment to a 50% reduction in the use of primary raw materials by 2030, formal commitment to this transition is taking shape. CircuLaw identifies opportunities in current law to support this circular future.</p>`